iquip/apps/scheduler.py

```diff
@@ -0,0 +1,307 @@
+"""App module for showing the experiment list."""
+
+from typing import Optional, Tuple
+
+from PyQt5.QtGui import QPainter
+from PyQt5.QtCore import Qt, QObject, QAbstractListModel, QModelIndex, QMimeData
+from PyQt5.QtWidgets import (
+    QStyleOptionViewItem, QWidget, QLayout, QLabel, QListView,
+    QPushButton, QHBoxLayout, QVBoxLayout, QAbstractItemDelegate, QAction
+)
+
+import qiwis
+from iquip.protocols import ExperimentInfo
+
+def delete_items_of_layout(layout: QLayout):
```

Comment:
"_of_layout" seems redundant since you already named the argument informative enough.

Reply:
To emphasize that this is an _auxiliary_ (helper) function, you can start its name with an underscore to denote that this is not a part of the public API of this module.
This is NOT mandatory!